Scripts/RemediationScripts/Readme.md

```diff
@@ -63,8 +63,14 @@ Bulk remediation scripts (BRS) can be used to remediate non-compliant resources/
 51. [Azure_CosmosDB_DP_Use_Secure_TLS_Version](Readme.md#51-Azure_CosmosDB_DP_Use_Secure_TLS_Version)
 52. [Azure_SynapseWorkspace_AuthN_SQL_Pools_Use_Microsoft_Entra_ID_Only](Readme.md#52-Azure_SynapseWorkspace_AuthN_SQL_Pools_Use_Microsoft_Entra_ID_Only)
 53. [Azure_VirtualMachineScaleSet_SI_Enforce_Automatic_Upgrade_Policy](Readme.md#53-azure_virtualmachinescaleset_si_enforce_automatic_upgrade_policy)
-54. [Azure_AppService_AuthN_FTP_and_SCM_Access_Disable_Basic_Auth](ReadMe.md#54-azure_appservice_authn_ftp_and_scm_access_disable_basic_auth)
-
+54. [Azure_AppService_AuthN_FTP_and_SCM_Access_Disable_Basic_Auth](Readme.md#54-azure_appservice_authn_ftp_and_scm_access_disable_basic_auth)
+55. [Azure_Subscription_Config_Enable_MicrosoftDefender_Databases](Readme.md#55-Azure_Subscription_Config_Enable_MicrosoftDefender_Databases)
+56. [Azure_Subscription_Config_Enable_MicrosoftDefender_ResourceManager](Readme.md#56-Azure_Subscription_Config_Enable_MicrosoftDefender_ResourceManager)
+57. [Azure_Subscription_Config_Enable_MicrosoftDefender_AppService](Readme.md#57-Azure_Subscription_Config_Enable_MicrosoftDefender_AppService)
```

Comment:
Redirection for this App service one is not working

Reply:
Fixed.